P13GoTo/P13GoTo/Program.cs

```diff
@@ -0,0 +1,159 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using System.Diagnostics;
+using System.Text;
+Console.OutputEncoding = Encoding.UTF8;
+
+int victoryScore = 0;
+int lostScore = 0;
+
+Random randomNumber = new Random(); //creates a random number generator
+
+Console.WriteLine("Hello there \n " +
+                  "(づ｡◕‿◕｡)づ \n " +
+                  "I have picked a random number between 1 and 100. It's your turn to guess it and get ready to guess wrong!");
+
+int numberOutput = randomNumber.Next(1, 101);
+
+bool keepPlaying = true; // controlS game loop
+
+while (keepPlaying)
+{
+    Console.WriteLine("Input number below");
+    int numberInput = Convert.ToInt32(Console.ReadLine());
+
+    if (numberInput > numberOutput)
+    {
+        Console.WriteLine(" (ó﹏ò｡) \n " +
+                          "Oof so close yet so far above. Try a lower number!");
+        lostScore++;
+        Console.WriteLine($"Correct guesses: {victoryScore}.  \n" +
+                          $"Wrong guesses: {lostScore}.");
+        
+        // Restart game
+        keepPlaying = true;
+        
+    } else if (numberInput < numberOutput)
+    {
+        Console.WriteLine("(⸝⸝ᴗ﹏ᴗ⸝⸝) ᶻ 𝗓 𐰁 \n " +
+                          "Are you even trying to guess it? Try a higher number.");
+        lostScore++;
+        Console.WriteLine($"Correct guesses: {victoryScore}.  \n" +
+                          $"Wrong guesses: {lostScore}.");
+        keepPlaying = true;
+        
+    } else if (numberInput == numberOutput)
+    {
+        victoryScore ++; 
+        Console.WriteLine("(⊙ _ ⊙ ) \n " +
+                          "Mighty impressed I am you guess the right amount. I demand a rematch!! \n " +
+                          $"Correct guesses: {victoryScore}. \n" +
+                          $"Wrong guesses: {lostScore}.");
+        
+        // Forces player to guess again 
+        keepPlaying = true;
```

Comment:
Technically, you don't need to set it to `true` all the time. It stays `true` until you purposefully set it to `false`.